test.js

```diff
@@ -93,7 +93,7 @@ describe('cpFile()', function () {
 
 	it('should not create dest on unreadable src', function (cb) {
 		cpFile('node_modules', 'tmp', function (err) {
-			assert(err.code === 'EISDIR');
+			assert(err.cause().code === 'EISDIR');
```

Comment:
`verror` allows one to chain Errors similar to [Java](http://docs.oracle.com/javase/tutorial/essential/exceptions/chained.html). It is, the new `VError` instance references the error which caused it, and sets an appropriate error message:
```
cannot copy from '/' to 'tmp': cannot read from '/': EISDIR, read
\---      CpyError      ---/   \-- CpFileError --/  \-- StreamError --/
```
So if `err.name === 'CpyError'`, we know that we already handled this error and only have to print the error message. Otherwise the error was thrown somewhere we have not imagined and we want to print the full error stack for debugging purposes.

I have not investigated `verror` in depth, but I think a library like this should be able to print the complete stack trace from CpyError to SreamError - I think that is the tricky part which can't be written in 2-3 loc.